LotkaVolterra/scenario_1.jl

```diff
@@ -147,7 +162,7 @@ savefig(pl_overall, joinpath(pwd(), "plots", "$(svname)_reconstruction.pdf"))
 # Generate the basis functions, multivariate polynomials up to deg 5
 # and sine
 b = [polynomial_basis(u, 5); sin.(u)]
-basis = Basis(b, u)
+basis = Basis(b,u)
```

Comment:
This is currently failing for me, so I couldn't test beyond this. Upto here everything works fine.

Reply:
Compare to the newer syntax https://datadriven.sciml.ai/dev/examples/10_noisy_nonlinear/

Reply:
The present syntax works fine with DataDrivenDiffEq v 0.8.2 . I created a new environment and added it there. Now I'm running this code there.

Reply:
try with v0.8.3 now

Reply:
Does this mean that it is computed but the error is in displaying the result?

```julia
julia> basis
Model ##Basis#294 with 23 Error showing value of type Basis:
ERROR: MethodError: no method matching Basis(::Vector{Equation}, ::Vector{Term{Real, Nothing}}, ::Vector{Any}, ::Vector{Any}, ::Vector{Any}, ::Num, ::Vector{Any}, ::DataDrivenDiffEq.var"#f#2", ::Symbol, ::Vector{Union{}}; checks=false)
Closest candidates are:
  Basis(::Any, ::Any, ::Any, ::Any, ::Any, ::Any, ::Any, ::Any, ::Any, ::Any) at C:\Users\user\.julia\packages\DataDrivenDiffEq\XTAjm\src\basis\type.jl:54 got unsupported keyword argument "checks"
  Basis(::Vector{Equation}, ::Vector, ::Vector, ::Vector, ::Vector, ::Num, ::Vector, ::Function, ::Symbol, ::Vector{Basis}) at C:\Users\user\.julia\packages\DataDrivenDiffEq\XTAjm\src\basis\type.jl:54 got unsupported keyword argument "checks"
  Basis(::AbstractVector{Equation}, ::AbstractVector; parameters, iv, controls, implicits, observed, name, simplify, linear_independent, eval_expression, kwargs...) at C:\Users\user\.julia\packages\DataDrivenDiffEq\XTAjm\src\basis\type.jl:111
```

Reply:
yes

Reply:
This should be fixed on DataDrivenDiffEq#master right now.